msgs.go

```diff
@@ -0,0 +1,218 @@
+package poa
+
+import (
+	"cosmossdk.io/core/address"
+	errorsmod "cosmossdk.io/errors"
+	"cosmossdk.io/math"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+var (
```

Comment:
denote which funcs & where there are in the SDK

(Don't delete code, only comment out)